src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+management.endpoints.enabled-by-default=false
+management.endpoints.web.base-path=/acsp-profile-data-api
+management.endpoints.web.path-mapping.health=healthcheck
+management.endpoint.health.show-details=never
+management.endpoint.health.enabled=true
+management.health.mongo.enabled=false
+
+api.api-url=${CHS_KAFKA_API_URL:http://localhost:8889}
+api.api-key=${ACSP_PROFILE_API_SUB_DELTA_KEY:chsApiKey}
+
+spring.data.mongodb.uri=${MONGODB_URL:mongodb://mongo:27017/acsp_profile_delta}
+spring.data.mongodb.name=acsp_profile_delta
```

Comment:
I think the database and collection should be "acsp_profile" instead, to be consistent with the company_profile database/collection name, "delta" is generally used for the Kafka topics.

Reply:
done